_regenerate_json.sh

```diff
@@ -68,20 +87,12 @@ done
     echo cf-secret
 ) | while read agent
 do
-    if [ -x "${WRKDIR}/core/${agent}/${agent}" ]; then
-        ${WRKDIR}/core/${agent}/${agent} --help > ${OUTDIR}/${agent}.help && echo "Extracted --help output from ${agent}"
-    elif [ -x "${WRKDIR}/nova/${agent}/${agent}" ]; then
-         ${WRKDIR}/nova/${agent}/${agent} --help > ${OUTDIR}/${agent}.help && echo "Extracted --help output from ${agent}"
-    else
-        echo "WARNING: No executable agent found at ${WRKDIR}/core/${agent}/${agent}"
-        echo "WARNING: No executable agent found at ${WRKDIR}/nova/${agent}/${agent}"
-    fi
+    myhelp ${agent} > ${OUTDIR}/${agent}.help
```

Comment:
I might instead call it `dumpHelp()` or `generateHelp()` or something a bit more descriptive than `myhelp`